android/src/main/java/org/mozilla/mozstumbler/client/mapview/tiles/CoverageOverlay.java

```diff
@@ -16,18 +16,27 @@
  * This class provides the Mozilla Coverage overlay
  */
 public class CoverageOverlay extends AbstractMapOverlay {
+    public static final String MLS_MAP_TILE_COVERAGE_NAME = "Mozilla Location Service Coverage Map";
     // Use a lower zoom than the LowResMapOverlay, the coverage can be very low detail and still look ok
-    private static final int LOW_ZOOM_LEVEL = 10;
+    public static final int LOW_ZOOM_LEVEL = 10;
+
+    public CoverageOverlay(final Context aContext, final String coverageUrl, MapView mapView) {
+        super(aContext);
+        setup(1, AbstractMapOverlay.LARGE_SCREEN_MIN_ZOOM, coverageUrl, mapView);
```

Comment:
I just noticed that [zoom level 0](https://d17pt8qph6ncyq.cloudfront.net/tiles/0/0/0.png) is used as an image on the MLS website frontpage. But it is not available currently, and it wasn't when I checked before. Just a temporary server side issue?